addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -690,7 +687,8 @@ registerModel({
          * @param {string} [reason]
          */
         async _onRecoverConnectionTimeout(token, reason) {
-            delete this._fallBackTimeouts[token];
+            const rtcSession = this.messaging.models['RtcSession'].insert({ id: token });
```

Comment:
note: this works by luck because identifying fields are encoded as string technically.

Because I think `id` is number and `token` is string.

Reply:
I can do type casting to make sure it's an int?
Eventually we won't deal with types at all, but until then type-cast is acceptable.

Reply:
It's fine, I'm just mentioning it. We'll very like clean the session/token code before changing the way identifying fields are done.